@here/harp-mapview/lib/MapView.ts

```diff
@@ -1921,8 +1934,16 @@ export class MapView extends THREE.EventDispatcher {
 
     /**
      * Requests a redraw of the scene.
```

Comment:
This might be a bit confusing, there is now a public method update which requests a redraw and a method redraw (which I guess does the same)